src/search/searchContainer.ts

```diff
@@ -0,0 +1,41 @@
+import { connect } from 'react-redux'
+import { ActionCreator } from 'redux';
+import { ThunkAction, ThunkDispatch } from 'redux-thunk'
+import { GET_COMPANY_ACTION } from '../companyReducer'
+import { getHistoryData } from '../history'
+import { getKeyStatsData } from '../keystats'
+import { getNewsData } from '../news'
+import { getOverviewData } from '../overview'
+import { getPeersData } from '../peers'
+import { API, GlobalState } from '../types'
+import { Company } from '../types'
+import Search from './search'
+
+export interface SearchContainerAction { type: string, company: Company }
```

Comment:
`type: typeof GET_COMPANY_ACTION`

Reply:
Call it something like `GetCompanyAction`? `SearchContainerAction` doesn't really explain what it does.